test/linter/test-real-values.js

```diff
@@ -76,6 +76,33 @@ function checkRealValues(supportData, blockList, relPath, logger) {
   }
 }
 
+/**
+ * @param {Identifier} data
+ * @param {string} [relPath]
+ */
+function findSupport(data, category, logger, relPath) {
+  for (const prop in data) {
+    if (prop === '__compat' && data[prop].support) {
+      if (blockList[category] && blockList[category].length > 0)
+        checkRealValues(
+          data[prop].support,
+          blockList[category],
+          relPath,
+          logger,
+        );
+    }
+    const sub = data[prop];
+    if (typeof sub === 'object') {
+      findSupport(
+        sub,
+        category,
+        logger,
+        relPath ? `${relPath}.${prop}` : `${prop}`,
+      );
+    }
+  }
+}
+
 /**
  * @param {string} filename
  */
```

Comment:
Wouldn't hurt to add a `@returns` annotation at the same time.

```suggestion
 * @returns {boolean}
 */
```